packages/wrangler/src/__tests__/pages/deployment-list.test.ts

```diff
@@ -0,0 +1,85 @@
+import { rest } from "msw";
+import { endEventLoop } from "../helpers/end-event-loop";
+import { mockAccountId, mockApiToken } from "./../helpers/mock-account-id";
+import { msw } from "./../helpers/msw";
+import { runInTempDir } from "./../helpers/run-in-tmp";
+import { runWrangler } from "./../helpers/run-wrangler";
+import type { Deployment } from "./../../pages/types";
+
+describe("deployment list", () => {
+	runInTempDir();
+	mockAccountId();
+	mockApiToken();
+
+	beforeEach(() => {
+		// @ts-expect-error we're using a very simple setTimeout mock here
+		jest.spyOn(global, "setTimeout").mockImplementation((fn, _period) => {
```

Comment:
Would it be worth breaking this out into a shared helper function so we don't have to copy/paste to every file?

Reply:
✅